modules/effects/src/concat_latest_from.ts

```diff
@@ -12,6 +12,18 @@ export function concatLatestFrom<T extends Observable<unknown>, V>(
  * `concatLatestFrom` combines the source value
  * and the last available value from a lazily evaluated Observable
  * in a new array
+ *
+ * ## Example
+ * Select the active customer from the NgRx Store
+ *
+ * ```ts
+ * import { concatLatestFrom } from '@ngrx/effects';
+ * import * fromCustomers from '../customers';
+ *
+ * this.actions$.pipe(
+ *  concatLatestFrom(() => this.store.select(fromCustomers.selectActiveCustomer))
```

Comment:
could be worth adding an example that shows that you can use the value from the stream in the argument list of the function passed into contactLatestFrom, too

like
`concatLatestFrom(action => this.store.select(fromCustomers.selectCustomer(action.id))`

Reply:
That's a good idea, thanks!